### PR TITLE
3d secure

### PR DESCRIPTION
**Pull request by muzafarova**
https://ubio-automation.atlassian.net/browse/SDK-63

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -157,13 +157,42 @@ function afterSdkInitiated(sdk, cacheConfig, local) {
     Summary.update();
 }
 
+let tdsTimeout;
```
Comment (qubyte) [inline src/main.js]: This module scoped variable worries me (effectively a global)

File: src/main.js [hunk 2]
```diff
@@ -157,13 +157,42 @@ function afterSdkInitiated(sdk, cacheConfig, local) {
...
+function handle3ds(sdk, event) {
+    if (event === 'tdsStart') {
+        clearTimeout(tdsTimeout);
+
+        sdk.getActiveTds()
```
Comment (qubyte) [inline src/main.js]: Feel free to use async-await in the bundle. It's used elsewhere.

File: src/main.js [hunk 3]
```diff
@@ -157,13 +157,42 @@ function afterSdkInitiated(sdk, cacheConfig, local) {
...
+}
+
 function addTracker(sdk) {
     let loading = false;
 
     const stop = sdk.trackJob((event, error) => {
         console.log(`event ${event}`);
 
         switch (event) {
+            case 'tdsStart':
```
Comment (qubyte) [inline src/main.js]: tdsTimeout can probably be placed in this function somewhere.

File: templates/hotel-booking/sections/final-price-consent.js [hunk 4]
```diff
@@ -9,19 +9,21 @@ export default (name, { selectedRooms, finalPrice }, skip, sdk) => {
...
+                    id="submit-btn-${name}">Confirm and pay</button>
             </div>
         `;
 
-        const modal = templates.modal(template, 'Price check', {
-            closeOnOverlay: false,
-            showClose: false
-        });
-
+        const modal = templates.modal(template, 'Price check', true);
```
Comment (qubyte) [inline templates/hotel-booking/sections/final-price-consent.js]: The most from object parameters to primitive parameters has made calls to `modal` more difficult to understand (I have to find the definition to know what this `true` does).

File: templates/pet-insurance/sections/final-price-consent.js [hunk 5]
```diff
@@ -12,19 +12,21 @@ export default (name, { estimatedPrice = {}, finalPrice = {} }, skip, sdk) => {
     if (finalValue > estimatedValue) {
         //template to just display on modal
         const template = html`
-            <p>The final price has changed. and will be:</p>
-            <b class="large">${templates.priceDisplay(finalPrice.price)}</b>
-            <div class="section__actions field field-set">
-                <button type="button" class="button button--right button--secondary" id="cancel-btn">Cancel</button>
-                <button type="button" class="button button--right button--primary" id="submit-btn-${name}">Confirm and pay</button>
+            <p class="dim">The final price has changed. and will be:</p>
```
Comment (qubyte) [inline templates/pet-insurance/sections/final-price-consent.js]: Not from this PR, but this sentence looks weird.

Comment (qubyte) [inline templates/pet-insurance/sections/final-price-consent.js]: Not from this PR, but this sentence looks weird.

Comment (muzafarova) [reply templates/pet-insurance/sections/final-price-consent.js]: true, the dot shouldn't be there

**Pull request merged**